src/VideoToolbox/VTCompressionProperties.cs

```diff
@@ -507,9 +510,20 @@ public partial class VTCompressionProperties {
 		[Introduced (PlatformName.iOS, 13,0)]
 		[Introduced (PlatformName.TvOS, 13,0)]
 #endif
-		public VTAlphaChannelMode AlphaChannelMode {
-			get => VTAlphaChannelModeExtensions.GetValue (GetNSStringValue (VTCompressionPropertyKey.AlphaChannelMode));
-			set => SetStringValue (VTCompressionPropertyKey.AlphaChannelMode, value.GetConstant ());
+		public VTAlphaChannelMode? AlphaChannelMode {
```

Comment:
This is a breaking change, you can't make a value type (enum or struct) nullable :/

Reply:
Okay! So I will revert the changes to this property